package.json

```diff
@@ -100,6 +100,7 @@
     "@types/crypto-js": "^3.1.47",
     "@types/react-native-calendars": "^1.20.7",
     "amplitude-js": "5.7.0",
+    "async-storage": "^0.1.0",
```

Comment:
I guess u installed it by mistake. Remove it and reinstall `node_modules` to update `package-lock.json`

Reply:
done